ops/pebble.py

```diff
@@ -1314,7 +1314,17 @@ def __repr__(self):
 class NoticeType(enum.Enum):
     """Enum of notice types."""
 
+    CHANGE_UPDATE = 'change-update'
+    """Recorded whenever a change is updated, that is, when it is first
+    spawned or its status was updated. The key for change-update notices is
+    the change ID.
+    """
+
     CUSTOM = 'custom'
+    """A custom notice reported via the Pebble client API or ``pebble notify``.
+    The key and data fields are provided by the user. The key must be in
+    the format ``example.com/path`` to ensure well-namespaced notice keys.
```

Comment:
Do we validate this?

Reply:
Yeah, we validate it (in the Pebble server), so the workload will get an error if it tries to add a custom notice (`pebble notify`) with an invalid key. The [API does "reasonable effort" validation](https://github.com/canonical/pebble/blob/18c93cbc280de20c9e21d323fe3a8bbb340777a2/internals/daemon/api_notices.go#L32-L34) with a regex.